### PR TITLE
Style updates

### LINKED ISSUE #1: Link building is wrong on android
https://filecoinproject.slack.com/archives/C02EQ3ELFBQ/p1667221561511799?thread_ts=1667149614.848069&cid=C02EQ3ELFBQ

> I got a bug though: when you upload and it gives you the link back, the link it gave was https://.ipfs.dweb.link//bafybeif24lynjex2ywces2y6fez53qqkwaet6trkpyhmmury3loymezigy/rzw73z.jpg, which isn’t right

> I get the same bug pasting a CID into the “Open in browser” thing

### PR DESCRIPTION
**Pull request by cmunns**
We are looking great right now. Looking forward to review of this one. In addition to some edits made after the recording of video of simulator I made I've also added an intro screen pictured below.

NOTE: May need to look into transitions with the transparent background on views causing a bit of overlap in some cases.

Also closes #1 

https://user-images.githubusercontent.com/1189523/210454835-3b100faf-2f91-4be9-93f5-fa12dc35a66b.mp4

Screenshot 2023-01-03 at 2 55 16 PM

### CODE REVIEW

File: .env
Comment (meandavejustice) [inline .env:1]: We don't want to check in this token here

**Comment by meandavejustice** [comment]
image

> "Visit some things on the IPFS Network:"

should be on a new line.

File: ionic.config.json
Comment (meandavejustice) [inline ionic.config.json:3]: Are we sure about forcing ios styles by default? Even for android?

**Comment by meandavejustice** [comment]
image

background gradient is rendering a bit odd on android (home and upload pages)

**Comment by meandavejustice** [comment]
Found a couple of more issues when testing on device, they non style issues but should be fixed before we merge this pr to avoid breaking changes.

1. COPY CID button after file upload messages user SUCCESS but nothing is actually added to clipboard
2. Since `ipfs://` protocols do not work in chrome, the new uploaded file UI doesn't link correctly on android. If you try to click the new `ipfs://{cid}` link, it just does nothing.
3. shortcut links on settings page do not work

File: src/app.scss
Comment (yocontra) [inline src/app.scss:1]: Can we pull this font into local files and not use a web font? Otherwise if the phone is offline this won't load. Maybe there is a webpack plugin that does this automatically?

File: src/components/Shortcuts/index.tsx
Comment (yocontra) [inline src/components/Shortcuts/index.tsx:12]: Is this fragment wrapper needed since this component returns a single IonList?

File: src/pages/Settings/index.tsx
Comment (yocontra) [inline src/pages/Settings/index.tsx:19]: Can you pull this from the main nodes code at runtime, so this doesn't need to be hardcoded and updated every time the list in that file changes?

File: src/pages/Share/index.tsx
Comment (yocontra) [inline src/pages/Share/index.tsx:40]: A lot of these changes in the PR are quote style changes - if you run the `lint` command it should conform everything to our standard and remove a lot of these diffs.

**Review by yocontra — CHANGES_REQUESTED** [review]
Couple of minor changes needed but otherwise code LGTM

**Comment by cmunns** [comment]
Screenshot 2023-01-10 at 1 22 55 PM

**Review by meandavejustice — APPROVED** [review]

**Pull request merged**
